app/controllers/invoices_controller.rb

```diff
@@ -41,10 +41,9 @@ def create
   end
 
   def update
-    ActionController::Parameters.permit_all_parameters = true
-    params1 = ActionController::Parameters.new(date_of_issue: format_date_locale(params[:invoice][:date_of_issue]), paid_to_date: format_date_locale(params[:invoice][:paid_to_date]))
+    @invoice.date_of_issue = format_date_locale(params[:invoice][:date_of_issue])
+    @invoice.paid_to_date = format_date_locale(params[:invoice][:paid_to_date])
```

Comment:
I would suggest replace the params itself with formatted dates instead of setting attributes. Do it in `before_filter` action for only `create` and `update`